bsp/stm32/libraries/STM32MPxx_HAL/STM32MP1xx_HAL_Driver/Inc/stm32mp1xx_hal.h

```diff
@@ -28,6 +28,7 @@
 
 /* Includes ------------------------------------------------------------------*/
 #include "stm32mp1xx_hal_conf.h"
+#include <rtthread.h>
```

Comment:
Should we modify HAL code to add RT-Thread header file? Please check the reason.

Reply:
Yes, RT-Thread header file needs to be added for each HAL library in the STM32 series. If not do that, an error is reported when using the command ` scons -jx` : NULL is undefined.